assets/src/components/admin/admin_tables.tsx

```diff
@@ -385,6 +376,64 @@ const ElevatorV2ScreensTable = (): JSX.Element => {
           Filter: DefaultColumnFilter,
           FormCell: FormTextCell,
         },
+        {
+          Header: "Elevator ID",
+          accessor: buildAppParamAccessor("elevator_id"),
+          mutator: buildAppParamMutator("elevator_id"),
+          Cell: EditableCell,
+          disableFilters: true,
+          FormCell: FormTextCell,
+        },
+        {
+          Header: "Accessible Path Arrow",
+          accessor: buildAppParamAccessor("accessible_path_direction_arrow"),
+          mutator: buildAppParamMutator("accessible_path_direction_arrow"),
+          Cell: EditableCell,
+          disableFilters: true,
+          FormCell: FormTextCell,
+        },
+        {
+          Header: "Accessible Path Text",
+          accessor: buildAppParamAccessor("alternate_direction_text"),
+          mutator: buildAppParamMutator("alternate_direction_text"),
+          Cell: EditableCell,
+          disableFilters: true,
+          FormCell: FormTextCell,
+        },
+        {
+          Header: "Accessible Path Image",
+          accessor: buildAppParamAccessor("accessible_path_image_url"),
+          mutator: buildAppParamMutator("accessible_path_image_url"),
+          Cell: EditableCell,
+          disableFilters: true,
+          FormCell: FormTextCell,
+        },
+        {
+          Header: "Dot X",
+          accessor: (row) =>
+            row.app_params.accessible_path_image_here_coordinates.x,
+          mutator: (row, value) => {
+            const newRow = structuredClone(row);
+            newRow.app_params.accessible_path_image_here_coordinates.x = value;
+            return newRow;
```

Comment:
nit: If we want to start taking advantage of lodash/fp, we could use `fp.set` here (and below) to do the nested update and handle the clone implicitly.